### ISSUE TITLE
Entity data overflows hover card when value of property contains long word

### ISSUE THREAD

**Issue by matthew-white** [getodk/central#854]
From @dbemke at https://github.com/getodk/central/issues/670#issuecomment-2551357448:

After editing an entity with a long name, the hover card in submission detail page is broken (wrong width of property name, entered values are show beyond the hover card.
https://staging.getodk.cloud/#/projects/97/forms/trees_registration/submissions/uuid%3A2d68092f-0d00-4123-b98a-b87f8c19f9f8
![Image](https://github.com/user-attachments/assets/8b57cdac-a302-457f-8f03-0e6a30a7c4e6)

**Steps to reproduce:**
1. Go to the Entities "Data” tab.
https://staging.getodk.cloud/#/projects/97/entity-lists/trees/entities
2. Click "More”.
3. In the detail page click "Edit”.
4. In e.g. circumference_cm field enter a long name/number and update the entity.
5. In the "Activity” part click the hyperlink to the submission.
6.  Hover over the "Created entity” hyperlink.

**Comment by matthew-white** [comment]
It looks like the issue doesn't always happen when the value of an entity property is long. In getodk/central-frontend#1110, I tested the value "54.6029931 18.2311011 72.70000457763672 23.649", and I saw that it wrapped and was truncated as expected. It looks like the issue happens specifically when the value contains a particularly long word or number. It's more about long words than just long values.
